playbooks/byo/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: OpenShift health checks
+  hosts: OSEv3
```

Comment:
This playbook needs to follow the pattern of the other check playbooks, invoking a parallel `playbooks/common/openshift-checks/adhoc.yml` playbook and evaluating groups along the way. `openshift_version` chokes otherwise. Also at some point we'll want to switch from using BYO group names to the common group names in our `is_active` methods.